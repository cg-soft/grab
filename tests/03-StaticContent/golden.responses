-> ui/index.html
HTTP/1.1 200 OK
Content-Type: text/html
Date: date
Connection: keep-alive
Transfer-Encoding: chunked

<!doctype html>  

<!-- paulirish.com/2008/conditional-stylesheets-vs-css-hacks-answer-neither/ --> 
<!--[if lt IE 7 ]> <html lang="en" class="no-js ie6"> <![endif]-->
==
-> ui/index.html
HTTP/1.1 200 OK
Content-Type: text/html
Date: date
Connection: keep-alive
Transfer-Encoding: chunked

<!doctype html>  

<!-- paulirish.com/2008/conditional-stylesheets-vs-css-hacks-answer-neither/ --> 
<!--[if lt IE 7 ]> <html lang="en" class="no-js ie6"> <![endif]-->
==
-> ui/index.html
HTTP/1.1 200 OK
Content-Type: text/html
Date: date
Connection: keep-alive
Transfer-Encoding: chunked

<!doctype html>  

<!-- paulirish.com/2008/conditional-stylesheets-vs-css-hacks-answer-neither/ --> 
<!--[if lt IE 7 ]> <html lang="en" class="no-js ie6"> <![endif]-->
==
-> 404 not found
HTTP/1.1 404 Not found: ./static/404
Date: date
Connection: keep-alive
Transfer-Encoding: chunked

Not found: ./static/404
==
