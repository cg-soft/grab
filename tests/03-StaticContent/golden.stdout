Server running at http://127.0.0.1:port
GET ./static/index.html
GET ./static/index.html
GET ./static/index.html
GET ./static/404
